### ISSUE TITLE
no-mixed-operators is missing ++ and --

### ISSUE THREAD

**Issue by userpixel** [eslint/eslint#8063]
Was wondering why `++` and `--` are excluded from [no-mixed-operators](http://eslint.org/docs/rules/no-mixed-operators)?

In my config I'm allowing them ([no-plusplus](http://eslint.org/docs/rules/no-plusplus) is `"off"`) but I still want to disallow mixing `++` and `+` or `--` and `-` (in two separate groups) to warn for possible typos.
(I can do that with `**` and `*` since they are both supported).

When I try to use `++` or `--` I get this error:

```
Error: userpixel/config/chrome-extension.js:
	Configuration for rule "no-mixed-operators" is invalid:
	Value "++" must be an enum value.
	Value "--" must be an enum value.
Referenced from: .eslintrc.yml
    at validateRuleOptions (node_modules/eslint/lib/config/config-validator.js:109:15)
    at Object.keys.forEach.id (node_modules/eslint/lib/config/config-validator.js:156:13)
    at Array.forEach (native)
    at Object.validate (node_modules/eslint/lib/config/config-validator.js:155:35)
    at load (node_modules/eslint/lib/config/config-file.js:522:19)
    at configExtends.reduceRight.e (node_modules/eslint/lib/config/config-file.js:391:36)
    at Array.reduceRight (native)
    at applyExtends (node_modules/eslint/lib/config/config-file.js:362:28)
    at Object.load (node_modules/eslint/lib/config/config-file.js:529:22)
    at loadConfig (node_modules/eslint/lib/config.js:63:33)
```

**Comment by not-an-aardvark** [comment]
I assume this is because `++` and `--` are unary operators (they only apply to one expression), but every other operator checked by `no-mixed-operators` is a binary operator (applying to two expressions).

I'm confused about what code you're trying to disallow -- could you provide an example?

**Comment by userpixel** [comment]
```javascript
a = 1, b = 100
a++  + b       // 101, somewhat confusing but still ok if you know what you're doing
a = 1, b = 100
a+ ++b         // 102, somewhat confusing but still ok if you know what you're doing
a = 1, b = 100
a+++ b         // 101, very confusing
```

**Comment by nzakas** [comment]
From the docs:

> This rule checks BinaryExpression and LogicalExpression.

The purpose of this rule is to clarify how binary and logical operators relate to one another in an expression. Unary operators do not fit into either category.
